frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer.tsx

```diff
@@ -108,6 +108,11 @@ function MainNavbarContainer({
   useEffect(() => {
     if (bookmarks?.length !== orderedBookmarks?.length) {
       setOrderedBookmarks(bookmarks as any);
+    } else {
```

Comment:
What happened here is every attempt to iterate on the array generates an infinite loop which can be seen server-side.

I know useEffects can be fickle. I don't remember simply iterating a passed array creating infinite loops. If you know a way to work around this, please edit this PR.